src/readme.azure.md

```diff
@@ -5,8 +5,8 @@
 azure: true
 powershell: true
 license-header: MICROSOFT_MIT_NO_VERSION
-branch: resource-hybrid-profile-fix
-repo: https://github.com/Azure/azure-rest-api-specs/blob/$(branch)
+commit: resource-hybrid-profile-fix
```

Comment:
Could you explain under which circumstances we can accept a branch name as `commit`? Or the best input should always be commit id?